src/Microsoft.DotNet.Build.Tasks/PackageFiles/BuildTestsPipeline.targets

```diff
@@ -0,0 +1,39 @@
+<Project ToolsVersion="12.0" DefaultTargets="DownloadBlobsFromAzureTargets" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
The filename for this target (BuildTestsPipeline.targets) was derived when this applied only to the build pipeline.  Now that this is more general purpose, you can change the name, or move this code somewhere else in the dev workflow.